packages/core/src/lib/pagination/lines.mts

```diff
@@ -0,0 +1,80 @@
+import { breakLines, rotate } from '../utils.mjs';
+import { type Layout } from './layout.type.mjs';
+
+type Inputs<T> = {
+  items: readonly T[];
+  /** The width of a rendered line in characters. */
+  width: number;
+  /** Renders an item as part of a page. */
+  render: (layout: Layout<T>) => string;
```

Comment:
Let's `Prettify<Layout<T>>` (util from `@inquirer/types`) here so the exposed type is the raw object and not a named type.

Reply:
I'm not sure I understand this. Do you just want me to write this as `Prettify<Layout<T>>` instead of `Layout<T>` here?